app/controllers/api_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApiController < ApplicationController
+  skip_before_action :verify_authenticity_token
+     private
```

Comment:
`private` has an odd indentation here.

Reply:
I'll circle back and fix that and start using a linter as mentioned in one of your other comments.